Encountify.Android/CustomMapRenderer.cs

```diff
@@ -103,19 +117,20 @@ public View GetInfoWindow(Marker marker)
         async void OnInfoWindowClick(object sender, GoogleMap.InfoWindowClickEventArgs e)
         {
             Locations pinLocation = new Locations(e.Marker.Position.Latitude, e.Marker.Position.Longitude);
-            var distanceString = await DistanceCounter.GetFormattedDistance(pinLocation);
+            var distanceString = (await DistanceCounter.GetFormattedDistance(pinLocation)).Split(" ");
 
-            var distance = distanceString.Split(" ");
+            var distanceNumber = distanceString.Take(1).First();
+            var distanceMetric = distanceString.Skip(1).First();
```

Comment:
This doesn't seem like a good usage as you can just write distanceString[0], also Miglė did all the calculations for distance in her pull request.